packages/mint-components/src/components/tax-and-cash/sqm-banking-info-form/sqm-banking-info-form-view.tsx

```diff
@@ -0,0 +1,552 @@
+    height: "40px",
+    borderRadius: "50px",
+    background: "var(--sl-color-gray-200)",
+  },
+  InfoAlert: {
+    "&::part(base)": {
+      backgroundColor: "transparent",
+      borderTop: "none",
+      border: "none",
+    },
+
+    "&::part(message)": {
+      display: "flex",
+      alignItems: "center",
+      justifyContent: "flex-start",
+      padding: "10px",
+      height: "max-content",
+    },
+
+    "& sl-icon::part(base)": {
+      color: "var(--sl-color-yellow-500)",
+    },
+  },
+  InfoWarningIcon: {
+    height: "26px",
+    width: "26px",
+  },
+};
+
+const sheet = createStyleSheet(style);
+const styleString = sheet.toString();
+
+const vanillaStyle = `
+    :host{
+      display: block;   
+    }
+    * {
+       margin: 0;
+       padding: 0;
+       box-sizing: border-box;
+    }
+
+    & sl-select::part(label) {
+      font-size: var(--sl-font-size-small);
+      font-weight: 600
+    }
+
+    p {
+      line-height: 18px;
+      color: var(--sl-color-gray-800);
+      font-size: var(--sl-font-size-small);
+    }
+
+    /* this class is dynamically applied in controller */
+    .error-input::part(base) {
+      border: 1px solid var(--sl-color-danger-500);
+      border-radius: var(--sl-input-border-radius-medium);
+    }
+
+    .error-input::part(help-text) {
+      color: var(--sl-color-danger-500) !important;
+    }
+    
+  `;
+
+export const BankingInfoFormView = (props: BankingInfoFormViewProps) => {
+  const {
+    states,
+    states: { formState },
+    callbacks,
+    text,
+    refs,
+    slots,
+  } = props;
+
+  const { classes } = sheet;
+
+  const getLoadingSkeleton = (
+    checkedValue: "toBankAccount" | "toPayPalAccount" | undefined,
+    inputNumber?: number
+  ) => {
+    let skeletons = [];
+
+    const flexBoxStyle = {
+      display: "flex",
+      gap: "8px",
+    };
+
+    if (checkedValue === undefined) {
+      return (
+        <div style={{ ...flexBoxStyle, flexDirection: "column", gap: "16px" }}>
+          <div class={classes.SmallSkeleton} />
+          <div style={{ ...flexBoxStyle, flexDirection: "row" }}>
+            <div class={classes.CheckboxSkeleton} />
+            <div class={classes.SmallSkeleton} />
+          </div>
+          <div style={{ ...flexBoxStyle, flexDirection: "row" }}>
+            <div class={classes.CheckboxSkeleton} />
+            <div class={classes.SmallSkeleton} />
+          </div>
+        </div>
+      );
+    }
+
+    if (checkedValue === "toPayPalAccount") {
+      return (
+        <div style={{ ...flexBoxStyle, flexDirection: "column" }}>
+          <div class={classes.SmallSkeleton} />
+          <div class={classes.LargeSkeleton} />
+        </div>
+      );
+    }
+
+    for (let i = 0; i < inputNumber; i++) {
+      skeletons.push(
+        <div style={{ ...flexBoxStyle, flexDirection: "column" }} key={i}>
+          <div class={classes.SmallSkeleton} />
+          <div class={classes.LargeSkeleton} />
+        </div>
+      );
+    }
```

Comment:
This can be done functionally with `.map`

Reply:
Refactored the code